README.md

```diff
@@ -18,40 +18,74 @@ Some of these are tricky!  Don't trust your first instinct.
 
 **a)** `"1" == 1`
 
+True, this is because it is comparing the values so even though one is a string it is coercing the value to be true.
+
 **b)** `"1" === 1`
 
+False, because when using triple equals you are comparing the pure values and a string does not equal the number.
+
 **c)** `x == 'x'`
 
+When x is undefined it cannot equal the other
+
 **d)** `x == (x+'')`
 
 **e)** `'' == ' '`
 
+False because the quotes do not equal the same type.
+
 **f)** `x = true`
 
+True, because we are defining x and it is a value therefore true.
+
 **g)** `var x; x == 'undefined'`
 
+False, because undefined is considered not a number therefore it cannot be equal to not a number because there is no value.
+
 **h)** `'9'<'10'`
 
+ True, because we have the less than symbol and 9 is less than 10
+
```

Comment:
Correct that it is true, but not quite for this reason. In this case the characters are being valued individually against each other by javascript, until it finds a character of numeric value higher or lower than the other.